docs/schema.graphql

```diff
@@ -1,1060 +0,0 @@
-schema {
-  query: RootQueryType
-}
-
-type Account {
-  account_udts(input: AccountChildAccountUdtsInput = {page: 1, page_size: 20, sort_type: DESC}): [AccountUdt]
-  eth_address: String
-  id: Int
-  nonce: Int
-  script: Json
```

Comment:
What's the decision choice here to use JSON string as the type?

Reply:
all GraphQL Public Service API definition was according to database schema design, thus here use JSON as the type with their settlement. 

ref: https://github.com/nervosnet/godwoken_explorer/blob/e0a8010110a58fabb00b94458cf37bec20e246e1/lib/godwoken_explorer/account.ex#L18